### PR TITLE
[misc] Fix JSDocs, fix lint errors, clean up code

### LINKED ISSUE #50: [feature] Require JSDocs
### Is your feature request related to a problem? Please describe

JSDocs should be required https://jsdoc.app/about-getting-started.html

### Describe the solution you'd like

Add JSDocs as an eslint requirement and retroactively add in JSDocs to existing code

### Describe alternatives you've considered

N/A

### Additional context

N/A

### PR DESCRIPTION
**Pull request by brendacs**
### Purpose of pull request

**Related Issue**: resolves #50

Fix JSDocs, fix lint errors, clean up code

### Pull request checklist

- [x] Branch and PR are named correctly
- [x] Updated relevant documentation
- [ ] Tested with a tester bot
- [ ] Wrote unit tests for changes

### Testing instructions

Bot still works

### CODE REVIEW

File: src/bots/ama/constants.js [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-    return `${escapedBackticks}
+  return `${escapedBackticks}
 Event title: ${event.title}
 Date: ${event.date.toDateString()}${escapedBackticks}`;
-}
+};
 
-let getCommandsString = (subCommands) => {
-    let str = escapedBackticks + "\n";
+// let getCommandsString = (subCommands) => {
```
Comment (sirNikolai) [inline src/bots/ama/constants.js:24]: Removing blocks of code would be better than leaving commented out

File: src/bots/ama/models/Event.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import mongoose from 'mongoose';
 
-const EventSchema = mongoose.Schema({
-    title: String,
-    date: Date
-}, {collections: 'Event', autoCreate: true});
+const EventSchema = mongoose.Schema(
```
Comment (sirNikolai) [inline src/bots/ama/models/Event.js:3]: This seems inconsistent.

File: src/bots/ama/subcommands/addEvent.js [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
-  handle: async function(args) {
-    if (args.length < 2) {
-      client.message.channel.send(getStrings().insufficientArgumentsAddEvent);
-      return;
-    }
+const handler = async (args) => {
+  if (args.length < 2) {
+    client.message.channel.send(getStrings().insufficientArgumentsAddEvent);
+    return;
+  }
```
Comment (sirNikolai) [inline src/bots/ama/subcommands/addEvent.js:9]: nl

File: src/bots/ama/tests/addEvent.test.js [hunk 4]
```diff
@@ -1,49 +1,53 @@
...
-    let results = await EventModel.find({});   
+    let results = await EventModel.find({});
 
     expect(results.length).toEqual(0);
 
     expect(client.message.channel.send).toHaveBeenCalledWith(
       'Need to supply date (yyyy-mm-dd) and title of event\n' +
-      '_Example_: 2020-01-01 start of the greatest year ever');
+        '_Example_: 2020-01-01 start of the greatest year ever'
+    );
```
Comment (sirNikolai) [inline src/bots/ama/tests/addEvent.test.js:50]: this one's subjective but feels like a weird dangling ending

File: src/bots/ama/tests/listEvents.test.js [hunk 5]
```diff
@@ -1,58 +1,62 @@
...
 import mongoose from 'mongoose';
 import 'babel-polyfill';
 
-describe('adding Event', () => {
-  test('ama lists all events', async () => {  
-    await Event({
-        title: "Old 1",
-        date: new Date("2015-05-01T12:00:00Z")
-    }).save();
+// const expectedEventList = `__**Past Events**__
```
Comment (sirNikolai) [inline src/bots/ama/tests/listEvents.test.js:9]: What was the reason for everything being commented out?

**Review by sirNikolai — APPROVED** [review]
LGTM but there are a few questions

File: src/bots/ama/tests/listEvents.test.js (hunk 5 above)
Comment (sirNikolai) [reply src/bots/ama/tests/listEvents.test.js:9]: Never mind, ignore this. Will be fixed with #58

File: src/bots/ama/constants.js (hunk 1 above)
Comment (brendacs) [reply src/bots/ama/constants.js:24]: Thought you would need it for something in the future but yeah I can just remove

File: src/bots/ama/models/Event.js (hunk 2 above)
Comment (brendacs) [reply src/bots/ama/models/Event.js:3]: What is inconsistent?

File: src/bots/ama/tests/addEvent.test.js (hunk 4 above)
Comment (brendacs) [reply src/bots/ama/tests/addEvent.test.js:50]: It's generally the convention, it just auto-formatted

File: src/bots/ama/models/Event.js (hunk 2 above)
Comment (sirNikolai) [reply src/bots/ama/models/Event.js:3]: The schema appearance, but upon second glance I guess is fine

**Pull request merged**